external/config.json

```diff
@@ -1,6 +1,6 @@
 {
-    "version": "v3-deps-142",
-    "zip_file_size": "136974507",
+    "version": "v3-deps-143",
```

Comment:
In this code review we update some libraries to latest version.
I propose to split this code review to at least 2 parts:
1. Update only new libraries - check if build passed and it works properly 
2. Rewrite code if needed

Reply:
@PatriceJiang i think @crazyhappygame 's suggestion is good.

Reply:
OK, I will do it later.

Reply:
It does not make sense do it later. 
My point is that as first we should createPR 1 for new library only. Check if it works. 
Than create PR 2 with other changes

Reply:
I will close this PR and create 2 separate PR as you say, thx for your advice.